package.json

```diff
@@ -73,41 +73,41 @@
 	},
 	"devDependencies": {
 		"@ava/typescript": "^1.1.1",
-		"@rollup/plugin-typescript": "^5.0.1",
+		"@rollup/plugin-typescript": "^5.0.2",
 		"@sindresorhus/tsconfig": "^0.7.0",
-		"@types/chrome": "0.0.117",
-		"@types/codemirror": "0.0.96",
+		"@types/chrome": "0.0.122",
+		"@types/codemirror": "0.0.97",
 		"@types/copy-webpack-plugin": "^6.0.0",
-		"@types/firefox-webext-browser": "^78.0.0",
+		"@types/firefox-webext-browser": "^78.0.1",
 		"@types/jsdom": "^16.2.3",
 		"@types/mini-css-extract-plugin": "^0.9.1",
-		"@types/react": "^16.9.41",
+		"@types/react": "^16.9.45",
 		"@types/terser-webpack-plugin": "^3.0.0",
-		"ava": "^3.10.0",
+		"ava": "^3.11.1",
 		"chrome-webstore-upload-cli": "^1.2.0",
 		"copy-webpack-plugin": "^6.0.3",
-		"css-loader": "^3.6.0",
+		"css-loader": "^4.2.1",
 		"daily-version": "^2.0.0",
 		"dot-json": "^1.2.0",
 		"eslint-config-xo-react": "^0.23.0",
-		"eslint-plugin-react": "^7.20.3",
-		"eslint-plugin-react-hooks": "^4.0.5",
-		"jsdom": "^16.2.2",
+		"eslint-plugin-react": "^7.20.5",
+		"eslint-plugin-react-hooks": "^4.0.8",
+		"jsdom": "^16.4.0",
 		"mini-css-extract-plugin": "^0.9.0",
 		"npm-run-all": "^4.1.5",
-		"rollup": "^2.19.0",
+		"rollup": "^2.23.1",
 		"size-plugin": "^2.0.1",
 		"string-replace-loader": "^2.3.0",
 		"stylelint": "^13.6.1",
 		"stylelint-config-xo": "^0.19.0",
 		"terser-webpack-plugin": "^3.0.6",
-		"ts-loader": "^7.0.5",
+		"ts-loader": "^8.0.2",
 		"ts-node": "^8.10.2",
 		"type-fest": "^0.16.0",
-		"typescript": "^3.9.6",
-		"web-ext": "^4.3.0",
-		"web-ext-submit": "^4.3.0",
-		"webpack": "^4.43.0",
+		"typescript": "^4.1.0-dev.20200810",
```

Comment:
On my computer, since https://github.com/sindresorhus/refined-github/pull/3157 I started getting incorrect "file was updated" logs by TypeScript every few seconds.

This version seems to fix it